AwsCryptographicMaterialProviders/runtimes/python/doc/conf.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import os
+import re
+from datetime import datetime
+import toml
+
+VERSION_RE = re.compile(r"""__version__ = ['"]([0-9.]+)['"]""")
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+
+def get_release():
+    try:
+        with open('../pyproject.toml', 'r') as toml_file:
+            data = toml.load(toml_file)
+            return data['tool']['poetry']['version']
+    except FileNotFoundError:
+        return None
```

Comment:
i assume this is fine, but maybe leave a comment why we don't need to raise an error here?

Reply:
I think this was just cruft. Removed